src/test/java/com/example/universe/simulator/entityservice/utils/TestUtils.java

```diff
@@ -21,4 +22,18 @@ public static GalaxyDto buildGalaxyDtoForUpdate() {
 
         return result;
     }
+
+    public static PlanetDto buildPlanetDtoForAdd() {
+        return PlanetDto.builder()
+                .name("name")
```

Comment:
Should also have a reference to Star object: `.star(StarDto.builder().id(UUID.randomUUID()).build())`